includes/blocks/course-theme/class-lesson-actions.php

```diff
@@ -174,20 +174,18 @@ public function render( array $attributes = [] ): string {
 			}
 		} else {
 			$render_quiz_button          = false;
-			$complete_button_class       = 'is-primary';
 			$has_incomplete_prerequisite = ! Sensei_Lesson::is_prerequisite_complete( $lesson_id, $user_id );
 			$quiz_permalink              = Sensei()->lesson->get_quiz_permalink( $lesson_id );
 			$is_quiz_submitted           = Sensei()->lesson->is_quiz_submitted( $lesson_id, $user_id );
 			$is_pass_required            = Sensei()->lesson->lesson_has_quiz_with_questions_and_pass_required( $lesson_id );
 
 			if ( ! empty( $quiz_permalink ) && ! $is_quiz_submitted ) {
-				$render_quiz_button    = true;
-				$complete_button_class = 'is-secondary';
+				$render_quiz_button = true;
 			}
 
 			// Complete button.
 			if ( ! $is_pass_required ) {
-				$complete_lesson_button = $this->render_complete_lesson( $complete_button_class, $has_incomplete_prerequisite );
+				$complete_lesson_button = $this->render_complete_lesson( 'is-secondary', $has_incomplete_prerequisite );
```

Comment:
This method is private and is only used here, so I think we can remove this param.